pandas/tests/series/test_analytics.py

```diff
@@ -147,7 +147,7 @@ def test_empty_multi(self, method, unit):
         tm.assert_series_equal(result, expected)
 
     @pytest.mark.parametrize(
-        "method", ['sum', 'mean', 'median', 'std', 'var'])
```

Comment:
TODO: make sure I have coverage for this elsewhere... I don't remember why I removed it here.

Reply:
These are handled up above in `test_empty`.